app/assets/javascripts/arclight/arclight.js

```diff
@@ -1,5 +1,4 @@
 //= require arclight/collection_scrollspy
-//= require arclight/collection_navigation
```

Comment:
This needs to be restored.

Reply:
I've restored this with https://github.com/projectblacklight/arclight/pull/934/commits/dbb6821eb3e4e100beec5ac099591b1263bf06ad